cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/templates/slurm_external_slurmdbd.conf

```diff
@@ -0,0 +1,11 @@
+{# Adding comments at the beginning of each line is a trick to indent the template without affecting the output #}
+{##}# slurm_external_slurmdbd.conf is managed by external configuration.
+{##}# Do not modify.
+{##}# Please add user-specific slurmdbd configuration options in slurmdbd.conf
+{##}DbdHost={{ nlb_dns_name }}
+{##}StorageHost={{ dbms_uri | uri_host }}
+{##}StoragePort={{ dbms_uri | uri_port }}
+{##}StorageLoc={{ dbms_database_name }}
+{##}StorageUser={{ dbms_username }}
+{##}StoragePass=dummy
+{##}{% endif %}
```

Comment:
I dont see the start of the if condition

Reply:
Done